styles/tidal-web/catppuccin.user.less

```
@@ -0,0 +1,394 @@
+    --wave-color-solid-accent-dark: darken(@accent, 5%);
+    --wave-color-solid-accent-darker: darken(@accent, 8%);
+    --wave-color-solid-accent-darkest: darken(@accent, 9%);
+    --wave-color-solid-base-fill: @base;
+    --wave-color-solid-contrast-fill: @text;
+    --wave-color-solid-base-bright: @surface0;
+    --wave-color-solid-base-brighter: @surface0;
+    --wave-color-solid-base-brightest: @surface2;
+    --wave-color-solid-base-brighter-ultra-thick: @surface1;
+    --wave-color-solid-contrast-dark: @text;
+    --wave-color-solid-contrast-darker: @subtext0;
+    --wave-color-solid-contrast-darkest: @subtext1;
+    --wave-color-solid-rainbow-blue-fill: @blue;
+    --wave-color-solid-rainbow-green-brighter: lighten(@green, 8%);
+    --wave-color-solid-rainbow-green-dark: darken(@green, 5%);
+    --wave-color-solid-rainbow-green-darker: darken(@green, 8%);
+    --wave-color-solid-rainbow-green-darkest: darken(@green, 9%);
+    --wave-color-solid-rainbow-green-fill: @green;
+    --wave-color-solid-rainbow-orange-fill: @peach;
+    --wave-color-solid-rainbow-purple-fill: @mauve;
+    --wave-color-solid-rainbow-red-bright: lighten(@red, 2%);
+    --wave-color-solid-rainbow-red-brighter: lighten(@red, 3%);
+    --wave-color-solid-rainbow-red-dark: darken(@red, 6%);
+    --wave-color-solid-rainbow-red-darker: darken(@red, 8%);
+    --wave-color-solid-rainbow-red-darkest: darken(@red, 9%);
+    --wave-color-solid-rainbow-red-fill: @red;
+    --wave-color-solid-rainbow-yellow-bright: lighten(@yellow, 3%);
+    --wave-color-solid-rainbow-yellow-brighter: lighten(@yellow, 9%);
+    --wave-color-solid-rainbow-yellow-dark: darken(@yellow, 6%);
+    --wave-color-solid-rainbow-yellow-darker: darken(@yellow, 8%);
+    --wave-color-solid-rainbow-yellow-darkest: darken(@yellow, 9%);
+    --wave-color-solid-rainbow-yellow-fill: @yellow;
+    --wave-color-solid-special-bright: lighten(@peach, 3%);
+    --wave-color-solid-special-brighter: lighten(@peach, 9%);
+    --wave-color-solid-special-dark: darken(@peach, 6%);
+    --wave-color-solid-special-darker: darken(@peach, 8%);
+    --wave-color-solid-special-darkest: darken(@peach, 9%);
+    --wave-color-solid-special-fill: @peach;
+    --wave-color-solid-warning-bright: lighten(@red, 3%);
+    --wave-color-solid-warning-brighter: lighten(@red, 9%);
+    --wave-color-solid-warning-dark: darken(@red, 6%);
+    --wave-color-solid-warning-darker: darken(@red, 8%);
+    --wave-color-solid-warning-darkest: darken(@red, 9%);
+    --wave-color-solid-warning-fill: @red;
+
+    [class*="notification"] [class*="error"] {
+      background-color: @red;
+    }
+
+    [class*="notification"]
+      [class*="error"]
+      [data-test*="notification-message"],
+    [class*="notification"] [class*="error"] [class*="button"] {
+      color: @base !important;
+    }
+
+    [class*="notification"] [class*="error"] [class*="icon"] {
+      fill: darken(@red, 20%);
+    }
```

```suggestion
    [class*="notification"] [class*="error"] {
      background-color: fade(@red, 30%);
    }

    [class*="notification"]
      [class*="error"]
      [data-test*="notification-message"],
    [class*="notification"] [class*="error"] [class*="button"] {
      color: @text !important;
    }

    [class*="notification"] [class*="error"] [class*="icon"] {
      fill: @red;
    }
```

We could also try something like this.